build/common.project.props

```diff
@@ -15,7 +15,7 @@
     <NETCoreTargetFramework Condition="'$(DotNetBuildSourceOnly)' == 'true'">net9.0</NETCoreTargetFramework>
     <NETCoreLegacyTargetFramework>netcoreapp3.1</NETCoreLegacyTargetFramework>
     <NETCoreLegacyTargetFramework Condition="'$(DotNetBuildSourceOnly)' == 'true'">net9.0</NETCoreLegacyTargetFramework>
-    <NETCoreLegacyTargetFrameworkForSigning>netcoreapp5.0</NETCoreLegacyTargetFrameworkForSigning>
+    <NETCoreLegacyTargetFrameworkForSigning>net8.0</NETCoreLegacyTargetFrameworkForSigning>
 
     <!-- Target frameworks for class libraries-->
     <TargetFrameworksLibrary>$(NetStandardVersion)</TargetFrameworksLibrary>
```

Comment:
I'd love to do that, however, we probably need to do https://github.com/NuGet/Home/issues/13079 first. 

NS2.0 ilmerge works, but net5.0 was a lot harder. 

I did some work on that, but had an issue with keeping the NuGet.Build.Tasks.Pack.resources assemblies. 
@jeffkl was doing some related work too, so maybe he's attempt is better.

Reply:
[Show statistics per target framework](https://github.com/NuGet/NuGetGallery/issues/3450) is a popular ask in nuget.org. Here is a [view of the package downloads for NuGet.Packaging over the past 6 weeks](https://www.nuget.org/stats/packages/NuGet.Packaging?groupby=Version&groupby=ClientName&groupby=ClientVersion). I agree that this view does not provide details on which target framework a download was requested for; instead, it offers a bird's-eye view of downloads for each package version over the 6-week period. We may need to add some functionality on the client side so that nuget.org can display stats per target framework for each version.

In the linked issue description, you have mentioned the following. Hence, I think we should remove support for netstandard2.0 in the near future after addressing https://github.com/NuGet/Home/issues/13079 as Nikolche suggested.

> NuGet does not support signed packages with the netstandard2.0 binaries, so any apps targeting net5.0, net6.0, or net7.0 and referencing one of the above packages will get the netstandard2.0 binaries from the package, and lose support for signed packages.